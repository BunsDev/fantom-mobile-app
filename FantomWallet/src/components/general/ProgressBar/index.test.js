import React from 'react';
import { shallow } from 'enzyme';

import PropgressBar from './index';

test('renders correctly', () => {
  const wrapper = shallow(<PropgressBar completed={0.2} remaining={0.8} />);
  expect(wrapper).toMatchSnapshot();
});
