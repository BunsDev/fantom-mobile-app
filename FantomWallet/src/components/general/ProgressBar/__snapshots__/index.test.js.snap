// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderWidth": 0.1,
      "flexDirection": "row",
      "height": 7,
      "marginTop": 13.34,
      "width": 750,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgb(0,177,251)",
        "flex": 0.2,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgb(255,255,255)",
        "flex": 0.8,
      }
    }
  />
</View>
`;
